### DISCUSSION TITLE
Search mode "randomly" exits

### DISCUSSION THREAD

**Discussion by sgatu** [LazyVim/LazyVim#3340] [Q&A] [answered]
Hi,

I've been using NeoVim(with lazyvim config) for maybe 4-6 months, and I'm still learning, but there is some behavior that I can't explain why is like that and what's the purpose. The issue is that when I press / for searching it starts the searching mode and while I'm writing what I'm searching it randomly exists this mode and starts executing the commands I'm pressing. 

For example, I press / and then write "session", all good it find "session" in the file and when I press Enter it goes to the first result, next I can jump between matches with "n". That's great. Now I write "sassion", it finds none, and when I press Enter it gives me an error that no matches in the code were found. But if I write "sae" the moment I press the "e", it just exists the search mode and the next key I press is interpreted in normal mode as a command. 

I've made a screen capture to get a better explanation of the issue. 

https://gifyu.com/image/SfrfW

Maybe the behavior is intended and I'm doing something wrong, or something is broken in my config. I don't really know.

**Comment by dpetka2001** [comment, marked as answer]
This behavior comes from `flash.nvim`, but this specific setting was set to `disabled` by default, so it shouldn't be happening unless you have it explicitly set in your personal configuration. Maybe you're on an older version of `flash.nvim` before this change was made? Make sure to update your plugins first.

**Reply by sgatu** [reply]
Damn, I should update more frequently. Thank you very much.
